### ISSUE TITLE
fan integrations

### ISSUE THREAD

**Issue by Manbearpig92** [t0bst4r/home-assistant-addons#6]
Hi, 

i was reading that fans are supported entities but when i expose them over to samsung smart things it acts like a switch. just on off and no percentage control.

**Comment by t0bst4r** [comment]
Yes, most controllers do not support fans via matter. So I implemented them as switch as a workaround. 

But it should have level control to control speed
